orchestration/dagster_orchestration/hca_manage/dataset.py

```diff
@@ -150,7 +150,8 @@ class DatasetManager:
     def __post_init__(self) -> None:
         self.stewards = {
             "dev": {"monster-dev@dev.test.firecloud.org"},
-            "prod": {"monster@firecloud.org"}
+            "prod": {"monster@firecloud.org"},
+            "real_prod": {"monster@firecloud.org"},
```

Comment:
Unrelated to the overall PR, this makes the `dataset.py` tool functional in the "real" prod environment.